src/main/java/quickdocs/logic/QuickDocsParser.java

```diff
@@ -0,0 +1,73 @@
+package quickdocs.logic;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import quickdocs.model.QuickDocsModelManager;
+import seedu.address.logic.commands.patient.ConsultationCommands;
+import seedu.address.logic.commands.patient.PatientCommands;
+import seedu.address.model.patient.exception.PatientException;
+
+/**
+ *  QuickDocsParser will take in the commands the user entered
+ *  and handles the various operations involving the different modules
+ */
+public class QuickDocsParser {
+
+    private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
+
+    private QuickDocsModelManager modelManager;
```

Comment:
I do not think that a parse should contain a model. A parser should be part of the logic module and only does one thing which is to generate command objects from user input.